src/auro-banner.js

```diff
@@ -180,9 +180,9 @@ export class AuroBanner extends LitElement {
               </${this.headerTag}>`
             : html``
           }
-            <slot name="description" class="description"></slot>
+            <slot name="description" class="description body-default"></slot>
             <slot name="action" class="action"></slot>
-            <slot name="disclaimer" class="disclaimer"></slot>
+            <slot name="disclaimer" class="disclaimer body-xs"></slot>
```

Comment:
```suggestion
            <slot name="disclaimer" class="body-xs"></slot>
```
Do we still need this `disclaimer` class?

Reply:
Nevermind, I guess we still need it.